lib/hanami/middleware/body_parser.rb

```diff
@@ -1,5 +1,6 @@
-require 'hanami/middleware/body_parser/parser'
 require 'hanami/utils/hash'
+require_relative 'body_parser/class_interface'
+require_relative 'body_parser/parser'
```

Comment:
I feel like `require_relative` is nicer for requiring "neighbouring" file like this (i.e. ones that are closely situated to the current file, and within the same overall module or concept). AFAIK `require_relative` is also a little faster than a straight-up require. You happy for me to do it like this?